cmd/migrate/v1_0_00.go

```diff
@@ -0,0 +1,34 @@
+package migrate
+
+import (
+	"fmt"
+
+	"github.com/dymensionxyz/roller/cmd/utils"
+	"github.com/dymensionxyz/roller/config"
+	"github.com/dymensionxyz/roller/relayer"
+)
+
+type VersionMigratorV1000 struct{}
+
+func (v *VersionMigratorV1000) ShouldMigrate(prevVersion VersionData) bool {
+	return prevVersion.Major < 1
+}
+
+func (v *VersionMigratorV1000) PerformMigration(rlpCfg config.RollappConfig) error {
```

Comment:
No changes required on the rollapp itself? how it's state will change to use the tendermint IBC?

Reply:
the change to the rollapp itself is in the rollapp binary.